code/game/objects/items/paintkit.dm

```diff
@@ -36,12 +36,6 @@
 	name = "voidsuit modification kit"
 	desc = "A kit for modifying a voidsuit."
 	uses = 2
-	var/new_light_overlay
-
-/obj/item/kit/suit/inherit_custom_item_data(var/datum/custom_item/citem)
```

Comment:
~~Is the custom item system just deprecated now, or is this just unnecessary with the single-icon system?~~
Looks like it's unnecessary with the single-icon system.